docs/developer_docs/tutorials/started_ada.rst

```diff
@@ -0,0 +1,44 @@
+Getting Started with Ada
+---------------
+
+First, log in to the `Ada platform <https://ada.stfc.ac.uk/>`_
+using your STFC email and password. Once logged in, you can select
+*New Workspace > IMAT > Tomography Recommended* to create a new workspace.
+
+After the workspace has been created, you will see two windows: one for the
+online VS Code editor and the other for the Ada Workspace GUI.
+
+In the online VS Code editor, select the *hamburger menu (top left) > Terminal > New Terminal*,
+and download the Mantid Imaging source code using Git::
+
+    git clone https://github.com/mantidproject/mantidimaging.git
+
+Navigate into the cloned repository directory::
+
+    cd mantidimaging
+
+Load the Conda environment setup script by running::
+
+    source /etc/profile.d/conda.sh
+
+Then confirm that the ``mantidimaging`` and ``mantidimaging_nightly`` environments appear when you list Conda environments::
+
+    conda env list
+
+Activate one of the Mantid Imaging environments:
+
+- ``mantidimaging`` — matches the latest release version
+- ``mantidimaging_nightly`` — includes the latest updates from the main GitHub branch
+::
+
+    mamba activate <preferred-environment>
+
```

Comment:
For development its best to make a new environment, so that it has the dev tools (and does not have the `mantidimaging` package itself installed) 

```
python3 ./setup.py create_dev_env
```
This will make an environment called `mantidimaging-dev`